### PR TITLE
Feature/versionable command and error

### PR DESCRIPTION
**Pull request by rhyslbw**
This PR completes the task of making all objects we intent to persist versionable.

### CODE REVIEW

**Comment by DominikGuzei** [comment]
What's the status on this PR?

**Comment by rhyslbw** [comment]
I'm back from my holiday tonight, so will finish this tomorrow.

File: source/serializables/error.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
 // Make errors serializable
 Space.Error.mixin(Space.messaging.Ejsonable);
 Space.Error.type('Space.Error');
+
+// Make errors versionable
+Space.Error.mixin(Space.messaging.Versionable);
```
Comment (rhyslbw) [inline source/serializables/error.js]: @DominikGuzei Any ideas why a second mixin throws an error?

Comment (DominikGuzei) [inline source/serializables/error.js]: Yes, i just found out – it is a bug in `space:base`, will push a fix PR in a few minutes!

**Comment by rhyslbw** [comment]
Ok @DominikGuzei ready for merge

**Pull request closed without merging**